src/blocking/queue.rs

```diff
@@ -198,18 +199,21 @@ impl Queue {
             .ok()
     }
 
-    pub fn fetch_periodic_tasks(&self, error_margin_seconds: i64) -> Option<Vec<PeriodicTask>> {
+    pub fn fetch_periodic_tasks(
+        &self,
+        error_margin_seconds: StdDuration,
```

Comment:
I think the name is wrong. since it's duration

Reply:
I forgot to change it

Reply:
[Error management](https://github.com/ayrat555/fang/pull/67/commits/b3fb2ab6b20328a26099ca8cce2f9f79d38cfc84)